src/storybook/patterns/dropdown-in-modals/dropdown-in-modals.stories.mdx

```diff
@@ -0,0 +1,247 @@
+
+<Canvas>
+  <Story name="insideOverflowWithTransformContainer">
+    {() => {
+      const options = useMemo(
+        () => [
+          { value: "1", label: "Option 1" },
+          { value: "2", label: "Option 2" },
+          { value: "3", label: "Option 3" },
+          { value: "4", label: "Option 4" },
+          { value: "5", label: "Option 5" }
+        ],
+        []
+      );
+      const dialogRef = useRef();
+      const dialogStyle = { width: "300px", height: "200px", overflow: "auto" };
+      const secondDialogContent = (
+        <DialogContentContainer style={dialogStyle} ref={dialogRef}>
+          <Dropdown
+            placeholder="Dropdown inside a scrollable dialog"
+            options={options}
+            insideOverflowWithTransformContainer
+          />
+        </DialogContentContainer>
+      );
+      return (
+        <Dialog
+          ref={dialogRef}
+          content={secondDialogContent}
+          hideTrigger={Dialog.hideShowTriggers.CLICK_OUTSIDE}
+          showTrigger={Dialog.hideShowTriggers.CLICK}
+        >
+          <Button>Press me for dropdown inside dialog!</Button>
+        </Dialog>
+      );
+    }}
+  </Story>
+</Canvas>
+
+## 'popupsContainerSelector' prop
+
+For the most complex use cases, you can utilize our dropdown's "popupsContainerSelector" prop.
+This prop allows you to render all the dropdown's popovers in a different node in the DOM, as specified by the provided selector.
+
+To implement this solution and ensure proper display of all dropdown popovers, you need to define a few styles for the container selector, as shown in the following code example:
+
+```css
+.container {
+  top: 0; // Set the container and the beginning of the web page
+  right: 0; // Set the container and the beginning of the web page
+  position: "fixed"; // The container should not move when scrolling the web page
+  width: "100%"; // The container should cover the whole viewport
+  height: "100%"; // The container should cover the whole viewport
+  pointerevents: "none"; // The container should not block any event
+  zindex: 99999; // The container z index should be heigher from the z-index of the modal/dialog which contains the dropdown itself
+}
+.container * {
+  pointer-events: auto; // all container children should handle events as usual
+}
+```
+
+<!--- We are hiding the story from the docs block becasue it's not working well in this view because of the way stories render there -->
+
+This solution can be useful for multiselection dropdown with single line state, which contains multiple popovers (one for the dropdown menu and one for the selections which hidden under the counter).
+If you have defined the dropdown's popovers container styles as required, you can use it as demonstrated in the following example <Link href="/?path=/story/technical-patterns-display-dropdowns-inside-pop-overs--popups-container-selector" withoutSpacing>(click here for a live demo)</Link>
```

Comment:
Link is not working for me correctly
![image](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/0841d251-dedf-4c11-a6f7-24f1006c3bc6)

Reply:
No live demo anymore?